.github/workflows/config/spelling_allowlist.txt

```diff
@@ -291,8 +312,10 @@ unitarity
 unitary
 unoptimized
 upvote
+utils
```

Comment:
I don't think this is a word and therefore should probably not be added to the allowlist.